Add EU Digital Assembly blog post (#3)
NordIQuEst/dev: _posts/2024-05-02-EU-Digital-Assembly-2023.md

{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-lumi-puhuri-interconnection.png" alt="Diagram showing of the interconnection between QAL 9000, LUMI and Puhuri." caption="Interconnection between QAL 9000, LUMI and Puhuri. Credit: Dr. Miroslav Dobsicek" %}

Dr. Dobsicek's live demonstration showcased two crucial stages involved in this innovative application.  
 
At the outset, he demonstrated a live calibration measurement of the QAL 9000 processor. Quantum computing applications rely on incredibly precise control of quantum states.  

To achieve this level of precision, Dr. Dobsicek utilized QAL 9000’s cloud RESTful API interface to perform a live recalibration measurement called a π-pulse. This specific pulse drives the transition of qubits, the quantum bits that hold information, between the quantum states of 0 and 1. With this recalibration a noticeable better state discrimination has been achieved.

{% include figure image_path="/assets/images/eu-digital-assembly-qal9000-live-tuneup.png" alt="A Screenshot of the live tune up of QAL 9000." caption="A Screenshot of the live tune up of QAL 9000. Credit: Martin Ahindura" %}

> Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on12 out of the 25 total qubits available on the QAL 9000 chip. 

Reviewer: ```suggestion
Of note is that the calibration was performed concurrently on a significant portion of the processor, focusing on 12 out of the 25 total qubits available on the QAL 9000 chip. 
```

> The state discrimination has improved after the recalibration; Less blue dots, |1> state, we present in the red point-could. 

Reviewer: ```suggestion
The state discrimination has improved after the recalibration; fewer blue dots, |1> state, we present in the red point-cloud. 
```

Reviewer: I am not sure why there is a line in the page at this point.

Reviewer: It seems that in this case jekyll does not like `|1>` even with Kramdown+Mathjax. Could you say "the first excited state" or the excited state instead?

Reviewer 2: I could just do `|1>`

After the successful recalibration, Dr. Dobsicek demonstrated a basic machine 
learning algorithm tailored to classify points within a two-dimensional plane. The algorithm was based on the concepts described in [1]. 

> {% include figure image_path="/assets/images/eu-digital-assembly-quantum-neural-network.png" alt="Diagram showing A quantum neural network in QAL 9000 with a classical optimizer in LUMI." caption="A quantum neural network in QAL 9000 with a classical optimizer in LUMI. Credit: Dr. Miroslav dobsicek" %}

Reviewer: ```suggestion
{% include figure image_path="/assets/images/eu-digital-assembly-quantum-neural-network.png" alt="Diagram showing A quantum neural network in QAL 9000 with a classical optimizer in LUMI." caption="A quantum neural network in QAL 9000 with a classical optimizer in LUMI. Credit: Dr. Miroslav Dobsicek" %}
```
